### DISCUSSION TITLE
I have successfully installed windows 8 dev preview in Qemu and run it in V86 (OUTDATED, REFER TO #892)

### DISCUSSION THREAD

**Discussion by spetterman66** [copy/v86#801] [Show and tell]
![image](https://user-images.githubusercontent.com/69303239/216819315-b82aff49-a591-4654-be11-415fac2ef8d6.png)
2048MB ram, 128MB vram, ACPI on. Google Drive link in comments, also tested working Windows 8 build 8133, password v86

**Comment by Le4fff** [comment]
i want the dev build. can i get it or get a link to a site hosting it.

**Reply by spetterman66** [reply]
https://drive.google.com/file/d/1ujAh03OSqfcCQaLYxnPtadR3DQ0J1yDq/view?usp=sharing decompress and open in copy.sh/v86 as hard drive image, and give it 2048m ram and 128m vram, and do acpi on

**Reply by Le4fff** [reply]
Thanks so much!

**Reply by spetterman66** [reply]
no problem!

**Reply by CorruptionVR** [reply]
@spetterman66 
hey, what's the password?

edit - nevermind, password is v86

**Reply by spetterman66** [reply]
yes

**Comment by Quantamyt** [comment]
nice lmao

**Comment by 36479756** [comment]
can i get the iso so i can use the internet to install programs?

**Reply by spetterman66** [reply]
This is the outdated version, see https://github.com/copy/v86/discussions/892

**Reply by 36479756** [reply]
oh, ok then
